common/pulp/common/constants.py

```diff
@@ -1,6 +1,7 @@
 from types import SimpleNamespace
 
 
+#: All valid task states.
```

Comment:
Without including these little `#:` jobbers, the constants don't show up in the docs. I've never liked this syntax, but I like having the constants show up way more so I think we need to do it, and make sure it's called out in the doc-writing guide, should we write one, to do this.